hugo/layouts/report.html

```diff
@@ -205,32 +221,32 @@ <h2 class="mb-3">Location</h2>
     </div>
 
     <div class="col-12 col-lg-6">
-      <h2 class="mb-3">Characteristics</h2>
+      <h2 class="mb-3">Your Home's Top Characteristics</h2>
       <div class="table-responsive mb-3">
         <table class="table table-bordered characteristic-table">
           <tbody>
             <tr>
-              <td><strong>Class</strong></td>
-              <td>{{ .card.chars.char_class }}</td>
+              <td><strong>Property Type</strong></td>
+              <td>{{ .card.char_class_detailed }}</td>
```

Comment:
Did we want to do this as well in the comps chars table?

Reply:
Hmm, good question. I think it's OK to leave it for now. The comp char table and map are much more detailed, so I think it's appropriate to use the more technical forms of these attributes. But we can revisit this decision later on.